templates/zerver/help/add-custom-emoji.md

```diff
@@ -0,0 +1,58 @@
+# Add custom emoji
+
+If you are an administrator of a Zulip organization, you can add custom emojis
+to your organization for your organization members to use.
```

Comment:
This sentence contains a lot of **organization**. You should remove one. Sorry for not catching this in the last round.